assemblies/pentaho-cde-renderer/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.pentaho.ctools</groupId>
+    <artifactId>cde-plugin-assemblies</artifactId>
+    <version>8.1.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>pentaho-cde-renderer-assembly</artifactId>
+
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>feature</module>
+    <module>karaf</module>
+    <module>docker</module>
+  </modules>
+
+  <properties>
+    <karaf-maven-plugin.version>3.0.8</karaf-maven-plugin.version>
+    <karaf-maven-plugin.aggregateFeatures>false</karaf-maven-plugin.aggregateFeatures>
+
+    <assembly.tarLongFileMode>posix</assembly.tarLongFileMode>
+
+    <pentaho.version>${project.version}</pentaho.version>
```

Comment:
This property is not needed. Use the proper ones from https://github.com/pentaho/build-resources/blob/8.1.0.0/version.properties.